### PR TITLE
Implemented:Product Identifier in settings and made changes wherever product indentifier is displayed by fetching it from the productIdentifier store, done all this using dxp-component

### LINKED ISSUE #186: implement product identifier plugin
## What is the motivation for adding/enhancing this feature?

Similar to all other apps, we want to use our DXP plugin to let users select their preferred product identifier and that should be reflected across the app.

### PR DESCRIPTION
**Pull request by sanskar345**
### Related Issues

Closes #186

### Short Description and Why It's Useful

Added product identifier card in settings, made changes wherever product identifier is displayed.
Done all this using dxp-component.

### Screenshots of Visual Changes before/after (If There Are Any)

inventory

### Contribution and Currently Important Rules Acceptance

- [x] I have read and followed [contribution rules](https://github.com/hotwax/inventory-count#contribution-guideline)

### CODE REVIEW

File: src/views/Settings.vue [hunk 1]
```diff
@@ -116,6 +145,8 @@ import { Actions, hasPermission } from '@/authorization'
 import { UserService } from '@/services/UserService'
 import { hasError, showToast } from '@/utils';
 import { translate } from "@/i18n";
+import { useProductIdentificationStore } from '@hotwax/dxp-components';
+import Logo from '@/components/Logo.vue';
```
Comment (ymaheshwari1) [inline src/views/Settings.vue:149]: Remove unwanted imports.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/locales/en.json [hunk 2]
```diff
@@ -68,5 +68,12 @@
   "Variance reason": "Variance reason",
   "Variance updated successfully": "Variance updated successfully",
   "View": "View",
-  "You do not have permission to access this page": "You do not have permission to access this page"
+  "You do not have permission to access this page": "You do not have permission to access this page",
```
Comment (ymaheshwari1) [inline src/locales/en.json:71]: Arrange the changes in locale files alphabetically.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/Settings.vue (hunk 1 above)
Comment (sanskar345) [reply src/views/Settings.vue:149]: done

File: src/locales/en.json (hunk 2 above)
Comment (sanskar345) [reply src/locales/en.json]: done

**Comment by ymaheshwari1** [comment]
Closing as will be handled using reusable components.

**Pull request closed without merging**